packages/flutter/lib/src/material/time_picker.dart

```diff
@@ -136,77 +485,147 @@ enum _TimePickerHeaderId {
   colon,
   minute,
   period, // AM/PM picker
+  dot,
+  hString, // French Canadian " h " literal
```

Comment:
remove "Canadian"

Reply:
I'm surprised French France doesn't use it, but if it's really specific to fr-CA, I guess leave it. :-)